packages/archlinux/mondoo/PKGBUILD

```diff
@@ -0,0 +1,42 @@
+# Maintainer: Mondoo Inc <hello@mondoo.com>
+# Maintainer: Dominik Richter <dom@mondoo.com>
+# Maintainer: Patrick Münch <patrick@mondoo.com>
+#
+pkgname=mondoo
+orignalVersion="2.2.3"
+pkgver="${orignalVersion/-/_}"
+pkgrel=1
+pkgdesc="Mondoo Client CLI for the Mondoo Policy as Code Platform"
+url="https://mondoo.com"
+license=('custom')
+source=(
+    "https://releases.mondoo.com/mondoo/${orignalVersion}/mondoo_${orignalVersion}_linux_amd64.tar.gz"
```

Comment:
this isn't a real file:
```
❯ makepkg
==> Making package: mondoo 2.2.3-1 (Wed Jan 31 14:55:46 2024)
==> Checking runtime dependencies...
==> Checking buildtime dependencies...
==> Retrieving sources...
  -> Downloading mondoo_2.2.3_linux_amd64.tar.gz...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0   127    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
curl: (22) The requested URL returned error: 404
==> ERROR: Failure while downloading https://releases.mondoo.com/mondoo/2.2.3/mondoo_2.2.3_linux_amd64.tar.gz
    Aborting...
    ```

Reply:
File removed from generator, it's no needed.

Reply:
did you forget to push? It still fails because of it